static/js/tools/scatter/chart.tsx

```diff
@@ -51,6 +51,8 @@ interface ChartPropsType {
   yUnits?: string;
   placeInfo: PlaceInfo;
   display: DisplayOptionsWrapper;
+  xDate: string;
```

Comment:
could you clarify here that this is the requested date (and not the date from the data). or a comment for the interface that these are all requested chart params.